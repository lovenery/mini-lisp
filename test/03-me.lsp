(print-num (+ 1 2 3 4))
(print-num (- 2 1))
(print-num (* 1 2 3 4))
(print-num (/ 10 5))
(print-num (/ 3 2))
(print-num (mod 8 5))

(print-bool (> 1 2))
(print-bool (< 1 2))
(print-bool ( = (+ 1 1) 2 (/ 6 3) ))

