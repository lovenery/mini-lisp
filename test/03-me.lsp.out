10
1
24
2
1
3
#f
#t
#t
